.tlmgr-dep

```diff
@@ -50,5 +50,6 @@ tools
 txfonts
 url
 xcolor
+xkeyval
```

Comment:
Something has changed in TeXlive.  This is needed explicitly for the build to succeed.